dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace CopyObject
+{
+    public class CopyObject
+    {
+        // Specify your bucket region (an example region is shown).
```

Comment:
When referring to an AWS "region," "Region" must be capitalized on all mentions. "AWS Region" on first mention in each code file, "Region" on subsequent mentions.

Reply:
Fixed in all projects.